tests/test_picker.py

```diff
@@ -287,15 +290,19 @@ def test_constructor_state(config):
     await p.run()
     EXPECTED_LOGGER_CALLS = [
         call("Picker 'testing-muh-picker' is configured:"),
+        call('FILE_CATALOG_REST_TOKEN = fake-file-catalog-rest-token'),
         call('FILE_CATALOG_REST_URL = http://kVj74wBA1AMTDV8zccn67pGuWJqHZzD7iJQHrUJKA.com/'),
         call('HEARTBEAT_PATCH_RETRIES = 3'),
         call('HEARTBEAT_PATCH_TIMEOUT_SECONDS = 30'),
         call('HEARTBEAT_SLEEP_DURATION_SECONDS = 60'),
         call('LTA_REST_TOKEN = fake-lta-rest-token'),
         call('LTA_REST_URL = http://RmMNHdPhHpH2ZxfaFAC9d2jiIbf5pZiHDqy43rFLQiM.com/'),
         call('PICKER_NAME = testing-muh-picker'),
+        call('WORK_RETRIES = 3'),
         call('WORK_SLEEP_DURATION_SECONDS = 60'),
+        call('WORK_TIMEOUT_SECONDS = 30'),
         call('Starting picker work cycle'),
+        call('Asking the LTA DB for a TransferRequest to work on.'),
         call('Ending picker work cycle')
```

Comment:
Personally, I think checking for the config to be specific values is a bad idea.  If we ever change them, this test fails for no reason.  Also, in general I don't really care what the config is as long as the code does what it's supposed to do.

Reply:
This test isn't really written with specific values in mind, but rather that the configuration values are logged as provided when the component starts up. The configuration values that are used in the test are taken from the fixture on Line 43-58, except for one modified in the test itself.

I'm not opposed to having the test use a completely internal configuration (i.e.: don't use the fixture and so avoid being brittle with respect to changes there), but I do think that it is important to test that the component is logging all of its configuration values.

Reply:
A new test with a better name, a better description, and a locally created configuration object make it a little more clear what the test is doing and why.